packages/widget-chat/src/components/icon/index.js

```diff
@@ -0,0 +1,27 @@
+import React, {PropTypes} from 'react';
+
+import classNames from 'classnames';
+
+import styles from './styles.css';
+
+function Icon(props) {
+  let iconCode = ``;
+  if (props.type === `delete`) {
+    iconCode = `\uE0E7`;
+  }
+  return (
+    <div className={classNames(`icon-container`, styles.iconContainer)}>
+      <div className={classNames(`icon`, props.type, styles.icon)} onClick={props.handleOnClick} title={props.title}>
+        {iconCode}
+      </div>
+    </div>
+  );
```

Comment:
This seems overly complicated for just an icon. It's probably a good idea to split this into a very basic presentational icon component and then something a bit more complex to handling clicks, maybe an icon button component.
I'm a strong proponent of these types of components to really be completely presentational.